tests/metrics/test_metrics.py

```diff
@@ -327,10 +327,11 @@ def test_get_active_users_for_month(self):
         month_today = date_today.month
         date_before = date_today + relativedelta(months=-2)
         date_before = datetime.date(year_today, date_before.month, 1)
+
         dates_in = [
             datetime.date(year_today, month_today, 1),
             datetime.date(year_today, month_today, 15),
-            datetime.date(year_today, month_today, 30)
+            datetime.date(year_today, month_today, 30 if month_today != 2 else 28)
```

Comment:
@taimoor-ahmed-1 whats the need for this change?

Reply:
current month is February and tests were failing because there are only 28 days in Feb, so had to add a check over here.